yarn.lock

```diff
@@ -12,6 +12,13 @@ __metadata:
   languageName: node
   linkType: hard
```

Comment:
This doesn't get rid of the old version. See lines 8-13 here.

Reply:
Should be better now